src/layers/SelectionLayer.js

```diff
@@ -115,7 +132,15 @@ export default class SelectionLayer extends CompositeLayer {
           },
           selectedFeatureIndexes: [],
           data: EMPTY_DATA,
-          onEdit: ({ updatedData, editType }) => {
+          onEdit: (event) => {
+            if (!event) {
+              // A null event was recieved,
+              // so we want to select an empty array to clear any previous selection.
+              onSelect({ pickingInfos: [] });
+              return;
+            }
```

Comment:
I'd have a slight preference for returning a non-null event, with an explicit editType... something like `"clearSelection"`. The meaning of `null` is pretty obvious right now, but it might be less obvious in the future. I also worry that when we give meaning to `null` we make it harder to understand bugs if `null` is returned by mistake.

Reply:
Good point, I will add a new constant to represent the "clear" edit type.